src/pieces/counting.ts

```diff
@@ -0,0 +1,127 @@
+import { Client, MessageEmbed, TextChannel } from 'discord.js';
+import { DB, CHANNELS } from '@root/config';
+import { Counter } from '../lib/types/Counter';
```

Comment:
```suggestion
import { Counter } from '@lib/types/Counter';
```
Gotta use those aliases.